src/ScenarioPreview/ScenarioPreview.types.ts

```diff
@@ -0,0 +1,71 @@
+import { SxProps, Theme } from "@mui/material";
+
+import { Label } from "../LabelSelector/Label.types";
+
+/**
+ * Type of user associated with the scenario
+ */
+export type User = {
+  /**
+   * Name of the user
+   */
+  name: string;
+  /**
+   * Optional color for the user's avatar
+   */
+  color?: string;
+};
+
+/**
+ * The props needed to define a scenario preview
+ */
+export type ScenarioPreviewProps = {
+  /**
+   * Name of the scenario
+   */
+  name: string;
+  /**
+   * Link to open the current scenario
+   */
+  href: string;
+  /**
+   * Image URL of the scenario
+   */
+  image: string;
+  /**
+   * Description of the scenario
+   */
+  description: string;
+  /**
+   * Format of the scenario (e.g., ASAM OpenDRIVE)
+   */
+  format: string;
+  /**
+   * Format version of the scenario
+   */
+  formatVersion: string;
+  /**
+   * File object representing the scenario file
+   */
+  file: string;
+  /**
+   * Date of creation of the scenario in ISO-8601 format
+   */
+  createdAt?: string;
+  /**
+   * Optional user who created the scenario
+   */
+  user?: User;
+  /**
+   * Optional labels associated with the scenario
+   */
+  label?: Label[];
+  /**
+   * Label for the road, if applicable
+   */
+  roadLabel?: string;
```

Comment:
This should not be an optional prop and it should be name "roadName". The `roadHref` prop is also missing. Please refer to the type defined in the ticket [TD-3227](https://sce.myjetbrains.com/youtrack/issue/TD-3227)

```ts
type ScenarioPreviewProps = {
    name: string,
    description: string,
    href: string,
    image: string,
    format: string,
    formatVersion: string,
    file: string,
    roadName: string,
    roadHref: string,
    createdAt?: string,
    user?: string,
    label?: Label[],
    sx: SxProps<Theme>
}
```